bats_test/downloads/folder-downloads/error_outputs.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+
+# Support lib
+load '/home/node/packages/node_modules/bats-support/load.bash'
+# File methods. DEPENDS on SUPPORT lib.
+load '/home/node/packages/node_modules/bats-file/load.bash'
+# Assertions.  DEPENDS on SUPPORT lib.
+load '/home/node/packages/node_modules/bats-assert/load.bash'
+
+# My custom function to trigger the download
+load './download_folder.sh'
+
+DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" >/dev/null 2>&1 && pwd)"
+# This folder ID came from this Drive ID "c0c8ba1c-efc5-420d-a07c-a755dc67f6b2"
+MY_FOLDER_ID="dedace8e-9d10-455f-ac47-25336fd3117b"
+
+@test "Errors out if the path is non-existent" {
+    cd "${DIR}"
+    run download_folder "${MY_FOLDER_ID}" "/non/existent/path"
+
+    [ "${status}" -eq 1 ]
+    [ "${output}" == "Error: ENOENT: no such file or directory, stat '/non/existent'" ]
+}
+
+@test "Errors out if the path exists as a non-file" {
+    cd "${DIR}"
+    mkfifo "${DIR}/some_fifo"
+    run download_folder "${MY_FOLDER_ID}" "${DIR}/some_fifo"
+
+    [ "${status}" -eq 1 ]
+    [ "${output}" == "Error: The destination isn't a folder!" ]
+}
+
+@test 'Errors out if the folder ID is omitted' {
+    cd "${DIR}"
+    run download_folder
+
+    # FIXME: it should return error code 1
+    # [ "${status}" -eq 1 ]
+    [ "${output}" == "error: required option '-f --folder-id <folderId>' not specified" ]
```

Comment:
Can we change these for assert_output?

https://github.com/ztombol/bats-assert#assert_output

Reply:
Applied